src/ceph.in

```diff
@@ -849,7 +849,11 @@ def main():
         return 1
 
     if parsed_args.help:
-        return do_extended_help(parser, childargs)
+        return do_extended_help(parser, childargs, ("mon", ""), ' '.join(childargs))
```

Comment:
please use single quote, to be more consistent with the rest of this source file.